tuareg.el

```diff
@@ -583,11 +584,12 @@ do not perturb in essential ways the alignment are used.  See
 See `prettify-symbols-compose-predicate'."
   ;; Refine `prettify-symbols-default-compose-p' so as not to compose
   ;; symbols for errors,...
-  (and (prettify-symbols-default-compose-p start end match)
-       (not (memq (get-text-property start 'face)
-                  '(tuareg-font-lock-error-face
-                    tuareg-font-lock-interactive-output-face
-                    tuareg-font-lock-interactive-error-face)))))
+  (when (fboundp 'prettify-symbols-default-compose-p)
```

Comment:
Because I wasn't sure if the `fboundp` compiler chumminess extended to `and`, but I've now verified that it does, so I'll simplify the code. Thank you for teaching me something new!